lib/rules/web/admin_console/4.10/catalog.xyaml

```diff
@@ -9,7 +9,6 @@ delete_template_instance_action:
   action: submit_changes
 create_app_from_imagestream:
   action: goto_create_app_from_imagestream_page
```

Comment:
merge `wait_form_loaded` with `goto_create_app_from_imagestream_page` in goto.xyaml